### ISSUE TITLE
Improve the documentation & arrange the problems in categories

### ISSUE THREAD

**Issue by geekyd** [geekyd/Programming-Problem-sets#31]
Improve the documentation & arrange the problems in categories

- [ ] Update the document 
- [ ] Create categories of the problems & arrange the problems according to the category
- [ ] Add advanced categories like algorithms etc.

**Comment by kairoschris** [comment]
I tried to create separate levels but it seems like there are more advanced things in the earlier sections.

**Comment by geekyd** [comment]
Yes we need to arrange the problems in specific category according to the difficulty of the problems.
